### ISSUE TITLE
scoreboard not showing completely on Bedrock/Pocket Edition

### ISSUE THREAD

**Issue by Devilkings-shubhamos** [SkytAsul/BeautyQuests#306]
I have a Server On Aternos which also have GeyserMC which allows bedorck clients to join. Recently i have made a small quest and it works on java and bedrock both but the problem is the Scoreboard is incomplete on bedrock Version which makes it impossible to see  

**Software**
BeautyQuests version:1.0.2
Game version:1.20.1
Server version:1.19.4
Server type: Spigot
Are you using a protocol hack (like ViaVersion/Viarewind)?: Yes

**Comment by SkytAsul** [comment]
Screenshot please

**Comment by AceSkym** [comment]
I appear to be having the same issue on my server with Geyser/Bedrock players. Here is the screenshot that was provided. I've tested to see if other plugins are effecting the scoreboard and that doesn't seem to be the case. Additionally I've tried adding additional lines and adjusting the "length" in the .yml. Nothing seems to change the behavior, size or display of the scoreboard in any effective way.

BeautyQuests version:1.0.2
Game version:1.20.4
Server version:1.20.4
Server type: Paper
Are you using a protocol hack (like ViaVersion/Viarewind)?: Yes

![20240121_172005](https://github.com/SkytAsul/BeautyQuests/assets/63478761/5ff641b1-41ea-4d28-b5c6-1324d9a58edd)
